packages/devextreme/js/__internal/ui/html_editor/html_editor.ts

```diff
@@ -614,24 +636,26 @@ class HtmlEditor extends Editor<Properties> {
     }
   }
 
-  _optionChanged(args) {
-    switch (args.name) {
+  _optionChanged(args: OptionChanged<Properties>): void {
+    const { name, value, previousValue } = args;
+
+    switch (name) {
       case 'converter': {
-        this._htmlConverter = args.value;
+        this._htmlConverter = value as Properties[typeof name];
 
-        const { value } = this.option();
+        const { value: currentValue } = this.option();
 
-        this._processHtmlContentUpdating(value);
+        this._processHtmlContentUpdating(currentValue);
         break;
       }
       case 'value': {
-        this._processHtmlContentUpdating(args.value);
+        this._processHtmlContentUpdating(value);
+
+        const { value: currentValue } = this.option();
 
-        // NOTE: value can be optimized by Quill
```

Comment:
Why did u remove this note? Do you think it's obvious?

Reply:
[081e5880faafd04de7b57955e52c34b00540bbbb]